docs/reference/modules/asset.md

```diff
@@ -31,6 +32,38 @@ By default, `watch` is set to `true`. A truthy value will cause the application
 
 When the `APOS_UPLOADFS_ASSETS` environment variable is present, this optional property can be used to configure an `uploadfs` instance that differs from the one configured by the `attachment` module, allowing changes in where assets from the webpack build process are stored and how they are served. Full documentation for uploadfs can be found [here](https://www.npmjs.com/package/uploadfs).
 
+### `rebundleModules`
+
+The `rebundleModules` option allows for overridding the `bundles` properties passed into `webpack` at the individual module level, including modules add through npm. This option takes an object with module names, or module names with a suffix made up of a `:` and individual file name from the module `ui/src` folder, as properties. Each property takes the name of the bundle where the file should be added as a string.
```

Comment:
You can't rebundle arbitrary files, we need to stop using that word here. What you can do is rebundle the code that *used to go to a specific bundle* from a particular module. Or, you can rebundle all of the code from that module.

Reply:
Changed (hopefully for the better).